agent/pkg/builder/container/container_builder.go

```diff
@@ -400,41 +402,51 @@ func prepareImage(dc *DockerContainerBuilder) error {
 	return nil
 }
 
-func createNetwork(dc *DockerContainerBuilder) *string {
-	networkName := fmt.Sprintf("%s-network", dc.networkMode)
+func createNetworks(dc *DockerContainerBuilder) []string {
+	var networkIds []string
```

Comment:
networkIDs ?

Reply:
would you like to rename the variable or ?